components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -220,6 +221,18 @@ open class GleanInternalAPI {
             GleanInternalMetrics.clientId.generateAndSet()
         }
 
+        try {
+            val packageInfo = applicationContext.packageManager.getPackageInfo(
+                applicationContext.packageName, 0
+            )
+            GleanInternalMetrics.appBuild.set(packageInfo.versionCode.toString())
+            packageInfo.versionName?.let {
+                GleanInternalMetrics.appDisplayVersion.set(it)
+            }
+        } catch (e: PackageManager.NameNotFoundException) {
+            logger.error("Could not get own package info, unable to report build id and display version")
```

Comment:
nit: In similar situations in the app we throw `AssertionError` here since we never expect this to happen.

Reply:
Good point. Changing this.